src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList/TaskListDataGrid.tsx

```diff
@@ -306,22 +306,7 @@ const TaskListDataGrid: React.FC<TaskListDataGridProps> = ({
     return params.id === -1;
   };
 
-  return (
-    <DataGrid
-      columns={columns}
-      rows={rows}
-      isCellEditable={isCellEditable}
-      experimentalFeatures={{ newEditingApi: true }}
-      processRowUpdate={processRowUpdate}
-      pageSize={pageSize}
-      rowsPerPageOptions={[5, 10, 15, 100]}
-      onPageSizeChange={(newPageSize) => {
-        localStorage.setItem(tableRowCount, String(newPageSize));
-        setPageSize(newPageSize);
-      }}
-      sx={styles.datagrid}
-    />
-  );
+  return <></>;
```

Comment:
why did you change these?

Reply:
Also a misunderstanding, changed it back.